packages/components/src/templates/header.stories.ts

```diff
@@ -57,41 +57,166 @@ export const SampleA = {
               <sd-icon
                 title="Unsere weiteren Auftritte"
                 name="system/website"
-                library="global-resources"
                 class="text-xl absolute -ml-[4px]"
               ></sd-icon>
             </div>
           </sd-navigation-item>
           <sd-navigation-item>
             <div class="w-4 h-4 flex items-center">
-              <sd-icon
-                title="Suche"
-                name="system/magnifying-glass"
-                library="global-resources"
-                class="text-xl absolute -ml-[4px]"
-              ></sd-icon>
+              <sd-icon title="Suche" name="system/magnifying-glass" class="text-xl absolute -ml-[4px]"></sd-icon>
             </div>
           </sd-navigation-item>
           <sd-navigation-item>
             <div class="flex items-center gap-2">
-              <sd-icon name="system/profile" library="global-resources" class="text-xl"></sd-icon
-              ><span>Mein Depot</span>
+              <sd-icon name="system/user" class="text-xl"></sd-icon><span>Portfolio</span>
+            </div>
+          </sd-navigation-item>
+          <sd-navigation-item>
+            <div class="flex items-center gap-2">
+              <sd-icon name="system/lock-locked" class="text-xl"></sd-icon><span>Application</span>
+            </div>
+          </sd-navigation-item>
+          <!-- bottom-right-area end !-->
+        </div>
+      </div>
+    </sd-header>
+    <sd-drawer id="sampleA__drawer" placement="end">
```

Comment:
Please write constistent IDs, ideally always in kebap case.

Reply:
Updated